src/renderer/services/utils/account.ts

```diff
@@ -78,29 +78,20 @@ export const deleteBeaconNodeContainers = async (): Promise<void> => {
     );
 };
 
-export interface IExportStatus {
-    message: string;
-    level: Level;
-}
+export const saveValidatorData = (
+    exportPath: string,
+    publicKey: string,
+    name: string,
+    slashingDB?: Interchange,
+): void => {
+    // store Keystore
+    const keystorePath = path.join(getConfig(remote.app).storage.accountsDir, DEFAULT_ACCOUNT, `${publicKey}.json`);
+    const keystoreSavePath = path.join(exportPath, `keystore-${name}.json`);
+    fs.copyFileSync(keystorePath, keystoreSavePath);
 
-export const exportKeystore = (validator: IValidator): IExportStatus | null => {
-    const savePath = remote.dialog.showSaveDialogSync(remote.getCurrentWindow(), {
-        title: `Saving keystore for validator "${validator.name}"`,
-        buttonLabel: "Export",
-        filters: [{name: "Keystore", extensions: ["json"]}],
-        defaultPath: path.join(remote.app.getPath("home"), `${validator.publicKey}.json`),
-    });
-    // save keystore if destination selected
-    if (savePath) {
-        const keystorePath = path.join(getConfig().storage.accountsDir, DEFAULT_ACCOUNT, `${validator.publicKey}.json`);
-        const copyResult = copyFile(keystorePath, savePath);
-        return {
-            level: copyResult.success ? Level.INFO : Level.ERROR,
-            message: copyResult.success
-                ? `Successfully exported keystore for validator ${validator.name} to ${savePath}.`
-                : `Export failed: ${copyResult.message}.`,
-        };
+    // store slashing DB
+    if (slashingDB) {
+        const slashingDBSavePath = path.join(exportPath, `${name}-slashing-db.json`);
```

Comment:
putting date and time into slashing db file might be good idea

Reply:
export both files in same time, we need timestamp for it?

Reply:
I meant just adding timestamp to this file. Timestamp is irrelevant for keystore but quite relevant for slashing db